apps/preview/next/pages/showcases/Filters/FiltersPanel.tsx

```diff
@@ -412,7 +412,7 @@ export default function FiltersPanel() {
                         value={value}
                         checked={rating === value}
                         name="radio-rating"
-                        onChange={(event) => setRating(event.target.value)}
+                        onClick={() => (rating === value ? setRating(null) : setRating(value))}
```

Comment:
Comment:
Same here:

```suggestion
                        onClick={() => (setRating(rating === value ? null : value)}
```